### ISSUE TITLE
Audio CD not working

### ISSUE THREAD

**Issue by MBRG** [automatic-ripping-machine/automatic-ripping-machine#185]
### Description
Basically what happens is this: When I enter an Audio CD (I have tried various) into the drive, it is being identified as such. In the log I can see the track infos from CDDB and there is also a subfolder in "Music" being created, which will hold a cover (if found) and an m3u file. But that is all.

Ripping of the tracks does not work. It seems to instead loop. I let the prcoess run for some time and now have a log file with over 2500 lines, which basically repeats itself. E.g. I can find numerous entries where the cover is being looked for, or CDDB being queried.

In htop I can then also see various processes vor cdparanoia and a very, very high load (up to 40, while DVD encoding only went up to 1.5). This then drops down and after some time starts again.

I also get some errors in bash like
Emask 0x2 (HSM violation)
ata2.00: status: DRDY ERR

but nothing in the log file.

### Log file
[Contents of log files under /opt/arm/logs]

### Config file
[Contents of your /opt/arm/config file]

### Environment
I'm running this in testing mode inside Virtual Box (Win 10 Prof Host) on Ubuntu server 18.04 with 1 processor and 1 GB RAM. I cloned the current master branch three days ago.

**Comment by muckngrind4** [comment]
What version of ARM are you using?  Is it master or v2_master.  Also, can you try ripping directly with abcde to see if you get the same behavior?

**Comment by muckngrind4** [comment]
Sorry.  I just saw you are on master.  Can you try abcde manually to see if it behaves the same?

**Comment by MBRG** [comment]
Hey,
did not have much time today, but I quickly ran 'abcde' and looked like the rip worked ok. But I need to test this again more thoroughly.

**Comment by MBRG** [comment]
Hello,
I tested again.  I moved the identify.sh file to another folder to prevent auto discovery. Then I simply ran 'abcde' as root (not via sudo) and got a fully ripped CD with every title being encoded as flac file, a cover and an m3u file. All files were placed in /mnt/media/ARM/Media/Music/flac/ARTISTNAME/ALBUMNAME.


Now when I copy back the identify.sh to /opt/arm and insert the same CD again I have again the same phenomena as before. 

htop shows entries like this:
` 5318 root       32  12 37104  6088  2160 D  0.0  0.6  0:00.02 cdparanoia --never-skip=40 -d /dev/sr0 01 //abcde.9a0c9a0b/track01.wav`

And in the log file I get amongst other things this:
```
Executing customizable pre-read function... done.
Getting CD track info... Grabbing entire CD - tracks: 01 02 03 04 05 06 07 08 09 10 11
abcde: attempting to resume from //abcde.9a0c9a0b..
trying to get cover with glyrc for Die Form / Photogrammes
copying cover to target directory /mnt/media/ARM/Media/Music//flac/Die Form/Photogrammes
Querying the CD for audio tracks...
```
Which looks ok, but after some times loops and gives me the same output again.

I've attached the whole log file from /opt/arm/logs here: [sLYS07Zq6hTAuNaNoy5igBeO3WQ-.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/2120202/sLYS07Zq6hTAuNaNoy5igBeO3WQ-.log)
